### PR TITLE
Add GenCC

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

File: reference_data/management/commands/utils/update_utils.py [hunk 1]
```diff
@@ -85,8 +89,8 @@ def update_records(reference_data_handler, file_path=None):
         with open_file(file_path, open_mode) as f:
             header_fields = reference_data_handler.get_file_header(f)
 
-            for line in tqdm(f, unit=" records"):
-                record = dict(zip(header_fields, line.rstrip('\r\n').split('\t')))
+            for line in reference_data_handler.get_file_iterator(f):
+                record = dict(zip(header_fields, line if isinstance(line, list) else line.rstrip('\r\n').split('\t')))
```
Comment (ShifaSZ) [inline reference_data/management/commands/utils/update_utils.py:93]: Would it be better to do splitting only when the line is an instance of `str`?

File: ui/shared/components/panel/variants/VariantGene.jsx [hunk 2]
```diff
@@ -330,18 +356,8 @@ export const GeneDetails = React.memo((
...
-              <Label size="mini" color={OMIM_SECTION.color} content="OMIM" />
-            </Segment>
-            <Segment color={OMIM_SECTION.color}>
-              {omimDetails}
-            </Segment>
-          </OmimSegments>
-        )
-      )}
+      <br />
+      {getDetailSections(GENE_DISEASE_DETAIL_SECTIONS, gene, compact, labelProps)}
```
Comment (ShifaSZ) [inline ui/shared/components/panel/variants/VariantGene.jsx:360]: I could be an extra blank line if there is neither GenCC nor OMIM.

**Review by ShifaSZ — CHANGES_REQUESTED** [review]
Mostly looks awsome.

File: reference_data/management/commands/utils/update_utils.py (hunk 1 above)
Comment (ShifaSZ) [reply reference_data/management/commands/utils/update_utils.py:93]: How about this comment?

Comment (hanars) [reply reference_data/management/commands/utils/update_utils.py:93]: no its better this way

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**